app/code/Magento/Wishlist/Block/Customer/Wishlist.php

```diff
@@ -77,15 +89,67 @@ protected function _prepareCollection($collection)
         return $this;
     }
 
+    /**
+     * Retrieve Wishlist Product Items collection
+     *
+     * @return \Magento\Wishlist\Model\ResourceModel\Item\Collection
+     */
+    public function getWishlistItems()
+    {
+        $page = ($this->getRequest()->getParam("p")) ? $this->getRequest()->getParam("p") : 1;
+        $limit = ($this->getRequest()->getParam("limit")) ? $this->getRequest()->getParam("limit") : 10;
+        if ($this->_collection === null) {
+            $this->_collection = $this->_createWishlistItemCollection();
+            $this->_prepareCollection($this->_collection);
+        }
+        $this->_collection
+            ->setPageSize($limit)
+            ->setCurPage($page);
+        return $this->_collection;
+    }
+
     /**
      * Preparing global layout
      *
-     * @return void
+     * @return $this
      */
     protected function _prepareLayout()
     {
         parent::_prepareLayout();
         $this->pageConfig->getTitle()->set(__('My Wish List'));
+        $pager = $this->getLayout()
```

Comment:
Please add the pager block to layout xml

Reply:
done